types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import winston = require('winston');
```

Comment:
Because:

```
ERROR: 1:8  no-import-default-of-export-equals  The module 'winston' uses `export = `. Import with `import winston = require('winston')`. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-import-default-of-export-equals.md
```

🤷‍♀️